lib.sh

```diff
@@ -1144,6 +1144,13 @@ then
 fi
 
 CHECK_USERS=$(occ_command_no_check user:list --output=json | jq -r 'keys[]')
+
+if [[ $CHECK_USERS = *" "* ]]
+then
+    print_text_in_color "$IRed" "There are user existent, that contain spaces - unable to send notifications"
```

Comment:
@szaimen 

Green = Scucess
Cyan = Information
Red = Error

Yes, it's wrong to use spaces in usernames

Reply:
Okay, good to know!